components/gene-panels/api/src/main/java/org/phenotips/panels/GenePanel.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see http://www.gnu.org/licenses/
+ */
+package org.phenotips.panels;
+
+import org.phenotips.vocabulary.VocabularyTerm;
+
+import org.xwiki.stability.Unstable;
+
+import java.util.List;
+import java.util.Set;
+
+import org.json.JSONObject;
+
+/**
+ * An object containing methods for manipulating and displaying gene panels related data.
```

Comment:
This is uninformative, it describes technical things that can already be inferred (of course it's an object, and all objects contain methods for manipulating data), while it should say more about what a gene panel is. Something like this would be better:

> A Gene Panel links {@link #getPresentFeatures() a collection of observed symptoms} to {@link #getPhenotypesForGeneList() a list of genes} frequently associated with these symptoms, in descending order of relevance.